### PR TITLE
switch the build guide to gcc10.2-gdb9 (which works) and add Tiger Lake information

### PR DESCRIPTION
**Pull request by ujfalusi**
crosstool-ng fails to build with the gcc8.1-gdb8.1 configs and while there update the page for Tiger Lake as well.

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -185,7 +185,10 @@
 def setup(app):
 # add_stylesheet() was renamed to add_css_file() in sphinx 1.8 released
 # in September 2018. add_stylesheet() will be removed in sphinx 4.0
-   app.add_stylesheet("sof-custom.css")
+    try:
```
Comment (lgirdwood) [inline conf.py:188]: duplicate of previous PR ?

**Review by lgirdwood — CHANGES_REQUESTED** [review]
LGTM, just needs the stylesheet patch removed.

File: conf.py (hunk 1 above)
Comment (ujfalusi) [reply conf.py:188]: Sorry, yes, I have based it on PR #345 in order to verify the build, just a sec, I'll drop that patch.

Comment (ujfalusi) [reply conf.py]: Removed the sphinx 4.0 support patch from this PR.

**Review by lgirdwood — APPROVED** [review]

**Review by kv2019i — APPROVED** [review]
Thanks @ujfalusi !

**Review by marc-hb — APPROVED** [review]

**Comment by lgirdwood** [comment]
@deb-intel this is mostly technical version changes so I'll merge.

**Comment by deb-intel** [comment]
@lrgirdwo Thanks for the heads up. No worries.

**Pull request merged**